### PR TITLE
LPS-204810 Moving the responsibility for changing the zIndex to single select

### PR DESCRIPTION
**Pull request by igor-franca**
Related [ticket](https://liferay.atlassian.net/browse/LPS-204810)

### CODE REVIEW

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/SingleSelect.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.lfr_object-single-select {
```
Comment (danvb112) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/SingleSelect.scss:1]: Hey @igor-franca, according to the [BEM pattern](https://getbem.com/naming/), we should have two underscores here: 

`lfr__object-single-select`

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.tsx [hunk 2]
```diff
@@ -72,8 +72,7 @@ export default function ObjectDefinitionNodeHeader({
 					<ClayDropDownWithItems
 						items={dropDownItems}
 						menuElementAttrs={{
-							className:
-								'lfr-objects__model-builder-node-dropdown',
+							style: {zIndex: 1034},
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.tsx:75]: Hey @igor-franca, could you please format this style object please?

```tsx
style: {
    zIndex: 1034
},
```

**Review by danvb112 — APPROVED** [review]
Hey @igor-franca, good job! Just two suggestons :smile:

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @igor-franca, good job! Just two suggestons 😄

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
checking the tests

**Comment by igor-franca** [comment]
After analyzing the test results, I only found a single flaw:

`CreateObject#DisassociateDeletionManyToManyWillAllowDelete`

I ran this test locally on the branch of this PR and it passed successfully.

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by igor-franca** [comment]
ci:forward:force

**Pull request closed without merging**